cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedExpr.qll

```diff
@@ -3040,7 +3084,8 @@ class TranslatedDestructorsAfterThrow extends TranslatedElement, TTranslatedDest
       // And otherwise, exit this element with an exceptional edge
       not exists(this.getChild(id + 1)) and
       kind instanceof ExceptionEdge and
-      result = this.getParent().getExceptionSuccessorInstruction(any(GotoEdge edge))
+      not kind.(ExceptionEdge).isSEH() and
+      result = this.getParent().getExceptionSuccessorInstruction(any(GotoEdge edge), true)
```

Comment:
I don't think `true` is correct here as that would mean "this exception can be caught by an SEH handler" which is not true given that this element is generated to handle C++ exceptions

Reply:
ugh, copy pasta. Done